client/webserver/template.go

```diff
@@ -60,6 +61,9 @@ func newTemplates(folder, lang string, embedded bool) *templates {
 
 // filepath constructs the template path from the template ID.
 func (t *templates) filepath(name string) string {
+	if t.embedded { // slashes even on Windows, see embed pkg docs
+		return t.folder + "/" + name + ".tmpl"
```

Comment:
The general advice would be to use `path.Join` for embedded paths, instead of `filepath.Join`, but this is a very simple operation and `path.Join` is over complex for the job in this case.